blog/posts/2020-07-09-miragejs-graphql-tools.md

```diff
@@ -0,0 +1,198 @@
+---
+authorId: chuckcarpenter
+categories: 
+  - miragejs
+  - graphql
+date: '2020-07-09'
+slug: miragejs-graphql-tools
+title: Simplify using MirageJS with GraphQL
```

Comment:
We need to add `previousTitle`, `previousSlug` etc here. The next/previous links are unfortunately not automated.